### ISSUE TITLE
Helm v3 helm repo update cannot load repositories file [MAC]

### ISSUE THREAD

**Issue by Silvest89** [helm/helm#6297]
I am using helm 3 beta1
Somehow helm tries to load an empty path string?
```
helm repo update
Error: couldn't load repositories file (): open : no such file or directory
```

```
helm repo list
NAME        	URL
rook-release	https://charts.rook.io/release
```

**Comment by akashshinde** [comment]
@Silvest89 this issue is already been fixed here https://github.com/helm/helm/pull/6292

**Comment by hickeyma** [comment]
Closing as fixed in #6292

**Comment by alexandrsemak** [comment]
Hello,

hm i got the same with new beta1 release:

```
./darwin-amd64/helm version
version.BuildInfo{Version:"v3.0.0-beta.1", GitCommit:"f76b5f21adb53a85de8925f4a9d4f9bd99f185b5", GitTreeState:"clean", GoVersion:"go1.12.9"}
```

```
 ./darwin-amd64/helm repo update
Error: couldn't load repositories file (): open : no such file or directory
```
